src/components/NcListItem/NcListItem.vue

```diff
@@ -767,12 +781,34 @@ export default {
 			}
 		}
 	}
+	.list-item-content__details {
+		display: flex;
+		flex-direction: row;
+		justify-content: end;
+	}
+	&--one-line {
+		height: 50px;
+		padding: 10px;
+		margin: 2px;
+		.list-item-content__main {
+			display: flex;
+			justify-content: start;
+			gap: 12px;
+			min-width: 0;
+		}
+		.list-item-content__details {
+			display: flex;
+			flex-direction: row;
+			justify-content: end;
+		}
+	}
 
 	&__anchor {
 		display: flex;
-		flex: 1 0 auto;
+		flex: 1 0 10%;
 		align-items: center;
-		height: var(--default-clickable-area);
+		height: calc(var(--default-clickable-area) + 7);
```

Comment:
We cannot add `7` with unit like `px`

Reply:
this shouldnt be there at all, not sure what i was trying to do. sorry :)